source/mutable.d.ts

```diff
@@ -3,17 +3,21 @@ Convert an object with `readonly` keys into a mutable object. Inverse of `Readon
 
 This can be used to [store and mutate options within a class](https://github.com/sindresorhus/pageres/blob/4a5d05fca19a5fbd2f53842cbf3eb7b1b63bddd2/source/index.ts#L72), [edit `readonly` objects within tests](https://stackoverflow.com/questions/50703834), and [construct a `readonly` object within a function](https://github.com/Microsoft/TypeScript/issues/24509).
 
+@see [`SetMutable`](source/set-mutable.d.ts) – a type that can selectively set some keys as mutable.
+
 @example
 ```
 import {Mutable} from 'type-fest';
 
 type Foo = {
 	readonly a: number;
-	readonly b: string;
+	readonly b: readonly string[];
```

Comment:
I think the addition of `readonly string[];` to the example confuses things.

Can that maybe be in a second, more advanced example or as a caveat in the explanatory text?

As people who skim a test could otherwise interpret that this also changes `readonly` values, as why else would it be mentioned there?

Reply:
This was requested by Sindre, but I think we can explain ourselves in the example itself, highlighting what happens and why is that showcased.

Reply:
Huh, actually doesn't line 19 kinda explains the reasoning already?

```ts
mutableFoo.b[0] = '3'; //=> Error: Index signature in type 'readonly string[]' only permits reading.ts(2542).
```

Reply:
Added an additional clarifying comment.